App.js

```diff
@@ -57,13 +80,126 @@ const TabNavigator = createBottomTabNavigator({
     })
   });
 
+  const showAlert = ((title, body) => {
+    Alert.alert(
+      title, body
+    );
+  })
+
 const AppContainer = createAppContainer(TabNavigator);
 
 export default class App extends Component<Props> {
 
+  constructor(props) {
+    super(props)
+    firebase.messaging().subscribeToTopic("all");
+    console.log("Subscribed to topic");
+    console.log("Checking Status")
+    this.checkStatus();
+    let curr = new Date();
+
+    Object.keys(events).forEach((event) => {
```

Comment:
this doesn't appear to be necessary?

Reply:
It doesn't fire upon app startup, but 15 mins from app startup.